toolchain/syno-rtd1619b-7.2/Makefile

```diff
@@ -0,0 +1,12 @@
+TC_ARCH = rtd1619b
```

Comment:
Ah, this is the new arch, and finally with a decent kernel version!  Let's hope they end-up switching to an supported lts for x64